src/router.ts

```diff
@@ -202,6 +204,14 @@ const routes: RouteConfig[] = [{
             name: 'moonpay',
             props: { modal: true },
             meta: { column: Columns.DYNAMIC },
+        }, {
+            path: '/stake',
+            components: {
+                modal: StakeModal,
+            },
+            name: 'stake',
+            props: { modal: true },
+            meta: { column: Columns.ACCOUNT }, // TODO: investigate usage
```

Comment:
@sisou What would be the recommended value here?

Reply:
I believe we start staking always from the address view? In that case `Columns.ADDRESS` would be appropriate.

Reply:
The `column` is used only on mobile (thin) screens to move the view to the respective column. Most modals have `Column.DYNAMIC`, so they don't move the view when opened. But here I believe the address column is what we want to show. (Setting it to ACCOUNT) would always move the view over to the account column (center column) when the modal is opened, which is weird, as its opened on the address column (right column).